src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -815,7 +815,8 @@
         useConfig['jqueryui'] = { attach: 'jqueryui' };
         useConfig['chosen'] = { attach: 'chosen', deps: ['jquery'] };
         useConfig['pips'] = { attach: 'pips', deps: ['jqueryui'] };
-        useConfig['underscore'] = { attach: 'underscore' };
+        useConfig['underscore'] = { attach: '_', deps: ['underscore'] };
+        useConfig['underscore-min'] = { attach: '_', deps: ['underscore'] };
```

Comment:
Not sure why `underscore` has to be a dependency of `underscore` here, but if you don't do this underscore is defined as `3` in some plugins.